src/main/drivers/vtx_common.h

```diff
@@ -66,13 +66,15 @@ typedef enum {
 #define VTX_6705_POWER_OFF     1
 #define VTX_6705_POWER_25      2
 #define VTX_6705_POWER_200     3
+#define VTX_6705_POWER_MAX     3
```

Comment:
What is the purpose of `VTX_TRAMP_POWER_MAX`, `VTX_SA_POWER_MAX` and `VTX_6705_POWER_MAX`? They dont seem to be used anywhere.

The existing `VTX_TRAMP_POWER_COUNT`, `VTX_SMARTAUDIO_POWER_COUNT` and `VTX_RTC6705_POWER_COUNT` are part of the hard-coded fallback values that are used only when `USE_VTX_TABLE` is **not** defined. Furthermore, they are only used in their respective vtx implementations. Outside of specific drivers (and even in them in most cases), `VTX_TABLE_MAX_POWER_LEVELS` or when possible `vtxTablePowerLevels` should be used.

Reply:
OK, removed